specification/http/src/main/scripts/org/kaazing/specification/http/rfc7235/auth.tests/test.with.no.authorization.constraints.authorized/request.rpt

```diff
@@ -0,0 +1,8 @@
+connect http://localhost:8000/echo/
+connected
+
+write method "GET"
+write header "Authorization" "Basic am9lOndlbGNvbWU=" #login joe:welcome

```

Comment:
For positive tests, I would suggest always using joe:welcome (instead of test:test or admin:admin)